oracle/orchestrator/market_mapper.go

```diff
@@ -31,16 +31,23 @@ func (o *ProviderOrchestrator) listenForMarketMapUpdates(ctx context.Context) {
 			// Fetch the latest market map.
 			response := mmProvider.GetData()
 			if response == nil {
-				o.logger.Debug("market map provider returned nil response")
+				o.logger.Info("market map provider returned nil response")
 				continue
 			}
 
+			o.logger.Info("market map provider received update")
```

Comment:
This is going to repeat every N seconds, but doesn't really give us any further insight into how the system is running. Overall I'd say it's not a useful log.

Reply:
we can probably only log on actual updates.

Reply:
and move this to debug.